tests/ensure_provider_tests.py

```diff
@@ -120,7 +118,7 @@ def create_nm_playbook(test_playbook):
         EXTRA_RUN_CONDITION, ""
     )
     if extra_run_condition:
-        extra_run_condition = "{}{}\n".format(
+        extra_run_condition = "{0}{1}\n".format(
```

Comment:
Why is this needed? The test helper script does not need to support old python versions, does it?

Reply:
I rewrote this to avoid the warning